.github/workflows/ci.yml

```diff
@@ -45,8 +45,14 @@ jobs:
           sudo cp unicode.mapping /etc/apache2/
           sudo mkdir -p /var/cache/modsecurity
           sudo chown -R www-data:www-data /var/cache/modsecurity
+      - name: first check config (to get syntax errors)
+        run: sudo apachectl configtest
       - name: start apache with module
-        run: |
-          sudo systemctl restart apache2.service
-          sudo cat /var/log/apache2/error.log
-
+        run: sudo systemctl restart apache2.service
+      - name: Show httpd error log
+        if: always()
+        run: sudo cat /var/log/apache2/error.log
+      - name: Show mod_security2 audit log
+        if: always()
+        run: sudo cat /var/log/apache2/modsec_audit.log
```

Comment:
```suggestion
        run: |
          errors="$(grep "ERROR" /var/log/apache2/modsec_audit.log)"
          if [ -n "${errors}" ]; then
            echo "Found errors in modsec_audit.log"
            echo "${errors}"
            exit 1
          fi
```

Reply:
@marcstern, @theseion - this is an unresolved conversation - what do you want to do with this?

Reply:
Depends. Probably not necessary to check for anything and ok to simply dump the log.

Reply:
This is a generic way to display the audit log. It's especially useful if a test fail and you want to see which rule triggered (or not).
On top of that, we have to add specific searches for pattern associated to requests, to ensure they were blocked (with the right rule) or they were accepted. That will be included in the "run-time" checks we'll implement soon (I hope).